tools/build-stm-latest.sh

```diff
@@ -39,4 +39,5 @@ do_build pybv3-network PYBV3 MICROPY_PY_WIZNET5K=1 MICROPY_PY_CC3K=1
 do_build pybv10 PYBV10
 do_build pybv10-network PYBV10 MICROPY_PY_WIZNET5K=1 MICROPY_PY_CC3K=1
 do_build stm32f4disc STM32F4DISC
+do_build stm32f429disc STM32F429DISC
```

Comment:
I'm going to guess that they won't want this here. I run an hourly build of all of the known boards that aren't part of the normal build (to pick up any errors that might otherwise go unnoticed)

See: https://github.com/dhylands/micropython/blob/travis-master/.travis.yml

I'd be happy to add the STM32F429DISC board there.